.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Completed each of the following items before creating a PR
+
```

Comment:
There's nothing to fill out for coding standards, test cases and hopefully the ticket reference (since it should have already been done in the commit message). I suppose I could at least break out the "design information" bullet but that might be a pipe dream anyway. This template may be fluid for the first few weeks.